compiler/damlc/tests/src/VisualTest.hs

```diff
@@ -0,0 +1,111 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+module VisualTest
+   ( main
+   ) where
+import DA.Cli.Visual
+import Data.Either
+import qualified Test.Tasty.Extended as Tasty
+import qualified Test.Tasty.HUnit    as Tasty
+import qualified DA.Pretty as DAP
+import qualified Development.IDE.Types.Location as D
+import qualified DA.Daml.LF.Ast as LF
+import qualified Development.IDE.Core.API as API
+import qualified Development.IDE.Core.Rules.Daml as API
+import qualified Data.NameMap as NM
+import qualified Control.Monad.Reader   as Reader
+import Development.IDE.Core.API.Testing
+import System.Environment.Blank (setEnv)
+import Control.Monad.Except
+
+data ExpectedChoices = ExpectedChoices
+    { _cName :: String
```

Comment:
As I am not referring the name directly within the tests, I get warning so had to use `_`